server/filestore.go

```diff
@@ -639,6 +685,35 @@ func (fs *fileStore) StoreMsg(subj string, hdr, msg []byte) (uint64, int64, erro
 	return seq, ts, nil
 }
 
+// skipMsg will update message block for a skipped message. If first
```

Comment:
Can you rephrase starting at "If first.."? I can't understand that sentence.

Reply:
Yes will rephrase. If block is empty them this is just a metadata update, no placeholder records need to be written.